generic/tohil.c

```diff
@@ -4037,18 +4032,14 @@ tohil_python_return(Tcl_Interp *interp, int tcl_result, PyObject *toType, Tcl_Ob
     if (toType != NULL) {
         if (!tohil_check_toType(toType))
             return NULL;
-
         // toType/pt is a borrowed reference; do not decrement its reference count
         pt = (PyTypeObject *)toType;
-        toString = pt->tp_name;
     }
-    // printf("tohil_python_return called: tcl result %d, to=%s, resulObj '%s'\n", tcl_result, toString, Tcl_GetString(resultObj));
 
-    if (toType == NULL || STREQU(toString, "tohil.tclobj")) {
+    if (toType == NULL || pt==&TohilTclObjType) {
         return TohilTclObj_FromTclObj(interp, resultObj);
     }
-
-    if (STREQU(toString, "str")) {
+    if (pt==&PyUnicode_Type) {
```

Comment:
The original code has a blank line before each block, consistently. I know this is a style nitpick but it would be nice to keep them consistent and spaced out. It's more readable.